package.json

```diff
@@ -25,8 +25,7 @@
     "lint:commit": "commitlint --from $(git describe --tags --abbrev=0) --to HEAD --verbose",
     "lint:license": "license-check-and-add check -f license-check-config.json",
     "lint:license:fix": "license-check-and-add add -f license-check-config.json -r $(date +%Y)",
-    "postinstall": "rm -rf node_modules/codemirror/mode/rpm/changes",
-    "postpublish": "cp -r package.json ..",
+    "postpublish": "node tasks/postpublish.js",
```

Comment:
I just took a deeper look at this `postpublish` script and in fact `semantic-release` doesn't look to call it so I think we can just remove it without any effect